djstripe/migrations/0012_apikey.py

```diff
@@ -88,6 +88,7 @@ class Migration(migrations.Migration):
                                 regex="^(pk|sk|rk)_(test|live)_([a-zA-Z0-9]{24,34})"
                             )
                         ],
+                        unique=True,
```

Comment:
can you add this as a separate migration in case people following along with master?  (we can squash dev migrations before deploy release anyway)